packages/guides/resources/template/html/guides/body/code.html.twig

```diff
@@ -2,5 +2,10 @@
     {# see the RawDirective for where this is coming from; a refactor is desired to move this onto its own template / renderer #}
     {{ node.value|raw }}
 {%- else -%}
-<pre{% if node.classes %} class="{{ node.classesString }}"{% endif %}><code class="language-{{ node.language }}{{ node.startingLineNumber ? ' line-numbers' }}" {{- node.startingLineNumber ? ' data-start=' ~ node.startingLineNumber }}>{{ node.value }}</code></pre>
+    {% if node.caption %}
```

Comment:
Thinking out loud, shouldn't we wrap those elements in a div, so they are grouped?

Reply:
I am currently working on the themes anyway. We can do this in a next step

Reply:
For reference, the output from standard Sphinx is like this:

```html
<div class="literal-block-wrapper docutils container" id="id1">
    <div class="code-block-caption"><span class="caption-text">config/routes.php</span><a class="headerlink" href="#id1" title="Permalink to this code">¶</a></div>
    <div class="highlight-php notranslate">
       <div class="highlight">
          <pre><span></span><span class="x">$foo = &#39;bar&#39;;</span></pre>
       </div>
    </div>
</div>
```